### ISSUE TITLE
Docs on Migrating to v2 from Nebulex.Adapters.Dist.Cluster

### ISSUE THREAD

**Issue by bernardo-martinez** [cabol/nebulex#198]
Hello,

I am reading the docs on migrating to v2, but i do not see any reference on how to proceed with mentioned adapter. Can you suggest something?

thanks!

**Comment by cabol** [comment]
Hey! Yeah, I see it is not mentioned in the migration guide. But the adapter you want to use is `Nebulex.Adapters.Partitioned` (it replaces the old one `Nebulex.Adapters.Dist`).

**Comment by bernardo-martinez** [comment]
ok i will create an MR to the docs once i finish the migration. 

On the other hand, also i do not see (and not even searching in the code of previous versions) a reference to what should `generation_size` be updated to. I see a reference from a [comment of yours](https://github.com/cabol/nebulex/issues/53#issuecomment-515796288) that leads my to infer it refers to the cache size, so it can be updated to `max_size`, right?

**Comment by cabol** [comment]
Right, as you may have seen maybe, the []() uses a local backend adapter, which by default is [Nebulex.Adapters.Local](https://hexdocs.pm/nebulex/Nebulex.Adapters.Local.html#module-options), so yes, you can configure the max cache size with the option `:max_size`, you will find it in the docs. A configuration example may look like this:

```elixir
config :my_app, MyApp.PartitionedCache,
  keyslot: MyApp.PartitionedCache,
  primary: [
    gc_interval: :timer.hours(12),
    max_size: 1_000_000,
    allocated_memory: 2_000_000_000 # In case you want to set a memory limit too
  ]
```

Let me know if that helps, stay tuned, thanks!

**Comment by bernardo-martinez** [comment]
Hey, regarding this, the `ttl` unit option (of set/put) was moved from seconds to miliseconds, right?

**Comment by cabol** [comment]
Correct, the `ttl` is in milliseconds.
